src/coreclr/src/System.Private.CoreLib/CreateRuntimeRootILLinkDescriptorFile.targets

```diff
@@ -7,7 +7,7 @@
   <PropertyGroup>
     <_ILLinkRuntimeRootDescriptorFilePath>$(ILLinkTrimXml)</_ILLinkRuntimeRootDescriptorFilePath>
     <_NamespaceFilePath Condition=" '$(_NamespaceFilePath)' == '' ">$(MSBuildThisFileDirectory)..\vm\namespace.h</_NamespaceFilePath>
-    <_MscorlibFilePath Condition=" '$(_MscorlibFilePath)' == '' ">$(MSBuildThisFileDirectory)..\vm\mscorlib.h</_MscorlibFilePath>
+    <_MscorlibFilePath Condition=" '$(_MscorlibFilePath)' == '' ">$(MSBuildThisFileDirectory)..\vm\corelib.h</_MscorlibFilePath>
```

Comment:
_MscorlibFilePath  could be renamed, it's only reffed a few lines below.

Reply:
`_MscorlibFilePath` is passed as an argument to msbuild task that lives in mono/linker and would take multi-repo change to rename. I had it renamed originally, but then it looked inconsistent with the other arguments passed to the task.